### PR TITLE
fix time zone problem, unify the time zone with downstream

### PR DESCRIPTION
**Pull request by leoppro**
Issue：#839
### What problem does this PR solve? 
When the system time zone of drainer is different from the downstream database, the timestamp value will be parsed mistakenly.

### What is changed and how it works?
In drainer/arbiter/reparo, we parse time using UTC time zone, and set the session time zone of downstream database UTC.
And add a field in file and kafka protobuf to marks if the timestamp is in UTC.

### Check List 

Tests 

 - Unit test
 - Manual test

### CODE REVIEW

File: drainer/translator/kafka.go [hunk 1]
```diff
@@ -126,7 +126,7 @@ func deleteRowToRow(tableInfo *model.TableInfo, raw []byte) (row *obinlog.Row, e
 	columns := tableInfo.Columns
 
 	colsTypeMap := util.ToColumnTypeMap(tableInfo.Columns)
-	columnValues, err := tablecodec.DecodeRow(raw, colsTypeMap, time.Local)
+	columnValues, err := tablecodec.DecodeRow(raw, colsTypeMap, time.UTC)
```
Comment (suzaku) [inline drainer/translator/kafka.go]: This change is not backward compatible, right?

Comment (leoppro) [reply drainer/translator/kafka.go]: yes

Comment (suzaku) [reply drainer/translator/kafka.go]: OK, we should make sure to document the behavior change since the version including this change.

Comment (july2993) [reply drainer/translator/kafka.go]: don't suggest make such a breaking change for down-type = kafka & file
maybe add a field to indicate the time zone of time.Local ?

Comment (GregoryIan) [reply drainer/translator/kafka.go]: how to deal with this problem now @leoppro?

Comment (leoppro) [reply drainer/translator/kafka.go]: I will add a flag for `down-type = kafka & file` to avoid breaking compatibility

Comment (leoppro) [reply drainer/translator/kafka.go]: fixed,PTAL again @july2993

**Comment by leoppro** [comment]
PTAL again @july2993

**Comment by leoppro** [comment]
I will add test case for timestamp value

**Comment by GregoryIan** [comment]
please make sure this PR alive, otherwise I will close it

**Pull request closed without merging**